assets/javascripts/discourse/routes/admin-plugins-explorer.js.es6

```diff
@@ -4,19 +4,36 @@ export default Discourse.Route.extend({
   controllerName: "admin-plugins-explorer",
 
   model() {
-    const p1 = this.store.findAll("query");
+    const p1 = this.store.findAll("group");
     const p2 = ajax("/admin/plugins/explorer/schema.json", { cache: true });
-    return p1
-      .then(model => {
-        model.forEach(query => query.markNotDirty());
+    const p3 = this.store.findAll("query");
 
+    return p1
+      .then(groups => {
+        let group_names = {};
+        groups.forEach(g => {
```

Comment:
Another trivial suggestion but this could be one line. `groups.forEach(g => groupNames[g.id] = g.name)`

Reply:
@eviltrout Prettier is forcing it onto multiple line